index.html

```diff
@@ -1,20 +1,18 @@
-jessica Baker
-
-"About"
-My name is Jessica Baker. I am 30 years old. I live in Palatine Il
-with my 3 chrildren. I am a student at CTD school.My favorite subject
-is math. My hobbies are listening to music and taking my chrildren 
-out to fun places.
-
-"Experience"
--javascript
--linux
-
-"Connect"
-https://github.com/Jessica12492/intro-to-programming-dorado
-https://mail.google.com
-https://www.facebook.com
-
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Personal Portfolio Website</title>
+    <meta charset="UTF-8">
```

Comment:
With the code formatted, it's a little easier to see that you're missing the closing `</head>` tag. As a reminder, the `head` tag is supposed to contain data about the page, that isn't necessarily shown to the user. Examples, are `<style>` tags for adding CSS styles to the page, `<script>` tags for adding Javascript functionality to the page, `<meta>` tags for telling the browser more info about the HTML like what language it's in or what letters/characters to use (in your case `"UTF-8"` characters).  

Here in your page, you have to tags, `<title>` and `<meta>` that should altogether be surrouneded by an opening `<head>` tag and a closing `</head>` tag. Each HTML page should have a maximum of **ONE** **\<head> \</head>** block.

Reply:
More about the head tag here: https://www.w3schools.com/tags/tag_head.asp#:~:text=The%20element%20is%20a,scripts%2C%20and%20other%20meta%20information.

and in video form here: https://www.youtube.com/watch?v=roT8AvdhVfo